guides/WRITING-DOCUMENTATION.md

```diff
@@ -68,7 +68,7 @@ Supported flags:
   - `[NAME]` is the filename of the react example to be displayed
 
 - **`@responsive`** Renders breakpoint toggles for the markup example.
-- **`@status [NAME]`** Displays a status badge. Supported values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.
+- **`@status [NAME]`** Displays a status badge. Supported values: `Draft`, `Ready` and `Deprecated`.
```

Comment:
I think we should edit these, we've never used `Ready`.

Reply:
This would also be a good opportunity to update the comment on line 87 in `processKssSection`

Reply:
I'm not following this section? 

```HTML
<button class="ds-c-button {{modifier}}">Button label</button>
```